### PR TITLE
Call CreateSandbox

### PR DESCRIPTION
**Pull request by jmrodri**
Remove the need for creating an apb user and namespace externally.

### CODE REVIEW

**Comment by jmrodri** [comment]
Fixes Issue #17

**Comment by dymurray** [comment]
Awesome! Can't believe how easy this was after Shawn's PR got put in. Worked great for me.

**Comment by dymurray** [comment]
Only thing this is missing is to read the current-context so that user doesn't need to pass in `-p`. I can solve that in a different PR.

File: pkg/runner/runner.go [hunk 1]
```diff
@@ -77,19 +79,32 @@ func RunBundle(action string, ns string, args []string) {
 		"bundle-action":   action,
 		"bundle-pod-name": pn,
 	}
+
+	// TODO: using edit directly. The bundle code uses clusterConfig.SandboxRole
+	// which is defined by the template. So far we've been using edit.
+
+	runtime.NewRuntime(runtime.Configuration{})
+	targets := []string{ns}
+	serviceAccount, namespace, err := runtime.Provider.CreateSandbox(pn, ns, targets, "edit", labels)
```
Comment (dymurray) [inline pkg/runner/runner.go]: We should expose ability to use `admin` instead of `edit`. Some APBs I'm testing fail on `edit` sandbox.

Comment (jmrodri) [reply pkg/runner/runner.go]: Do you want that to be a config? What requirements would be needed to allow that? i.e. can any user have admin rights? I guess they can to their own namespaces.

**Comment by jmrodri** [comment]
> Awesome! Can't believe how easy this was after Shawn's PR got put in. Worked great for me.

Well since we can't create the target namespace using it, that simplified things. :)

**Comment by jmrodri** [comment]
### pass in namespace that doesn't exist
```
[jesusr@speed3 sbcli{remove-script}]$ ./sbcli bundle provision -p bastard  postgresql-apb
List of available plans:
name: dev
name: prod
Enter name of plan you'd like to deploy: dev
Plan: dev
Enter value for parameter [postgresql_database], default: [admin]: 
Enter value for parameter [postgresql_user], default: [admin]: 
Enter value for parameter [postgresql_password], default: [<nil>]: password
Enter value for parameter [postgresql_version], default: [9.6]: 
WARN Failed to create a InternalClientSet: unable to load in-cluster configuration, KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT must be defined. 
INFO OpenShift version: v3.10.0-alpha.0+b0ba202-1424 
WARN Failed to create a InternalClientSet: unable to load in-cluster configuration, KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT must be defined. 
INFO unable to retrieve the network plugin, defaulting to not joining networks - clusternetworks.network.openshift.io "default" is forbidden: User "dude" cannot get clusternetworks.network.openshift.io at the cluster scope: User "dude" cannot get clusternetworks.network.openshift.io at the cluster scope 

Problem creating sandbox [bundle-72dbd700-7bbf-47f7-88c2-4721d2a993ad] to run bundle. Did you run `oc new-project bastard` first?
```

**Comment by jmrodri** [comment]
### create namespace before running
```
$ oc new-project bastard
Now using project "bastard" on server "https://172.17.0.1:8443".

You can add applications to this project with the 'new-app' command. For example, try:

    oc new-app centos/ruby-22-centos7~https://github.com/openshift/ruby-ex.git

to build a new example application in Ruby.
```

### Now rerun provision

```
$ ./sbcli bundle provision -p bastard  postgresql-apb
List of available plans:
name: dev
name: prod
Enter name of plan you'd like to deploy: dev
Plan: dev
Enter value for parameter [postgresql_database], default: [admin]: 
Enter value for parameter [postgresql_user], default: [admin]: 
Enter value for parameter [postgresql_password], default: [<nil>]: password
Enter value for parameter [postgresql_version], default: [9.6]: 
WARN Failed to create a InternalClientSet: unable to load in-cluster configuration, KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT must be defined. 
INFO OpenShift version: v3.10.0-alpha.0+b0ba202-1424 
WARN Failed to create a InternalClientSet: unable to load in-cluster configuration, KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT must be defined. 
INFO unable to retrieve the network plugin, defaulting to not joining networks - clusternetworks.network.openshift.io "default" is forbidden: User "dude" cannot get clusternetworks.network.openshift.io at the cluster scope: User "dude" cannot get clusternetworks.network.openshift.io at the cluster scope 
INFO Creating RoleBinding bundle-d6188309-d1ba-4fb1-b73f-c7a960471592 
INFO Successfully created apb sandbox: [ bundle-d6188309-d1ba-4fb1-b73f-c7a960471592 ], with edit permissions in namespace bastard 
INFO Running post create sandbox functions if defined. 
Successfully created pod [bundle-d6188309-d1ba-4fb1-b73f-c7a960471592] to provision [postgresql-apb] in namespace [bastard]
```

**Pull request merged**